include/tvm/target/target_kind.h

```diff
@@ -364,7 +364,7 @@ inline TargetKindRegEntry& TargetKindRegEntry::set_attr(const String& attr_name,
 }
 
 inline TargetKindRegEntry& TargetKindRegEntry::set_device_type(int device_type) {
```

Comment:
should this change to set_default_device_type too?

Reply:
Done.